client/components/FailingAssertionsSummary/Table.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import { useFailingAssertions } from '../../hooks/useFailingAssertions';
+import { TestPlanReportPropType } from '../common/proptypes';
+import './FailingAssertionsSummary.css';
+
+const FailingAssertionsSummaryTable = ({
+  testPlanReport,
+  atName,
+  getLinkUrl = assertion => `#result-${assertion.testId}`,
+  LinkComponent = Link
+}) => {
+  const failingAssertions = useFailingAssertions(testPlanReport);
+  const { metrics } = testPlanReport;
+
+  if (failingAssertions.length === 0) return null;
+
+  return (
+    <>
+      <p>
+        {failingAssertions.length} assertions failed across{' '}
+        {metrics.mustAssertionsFailedCount +
+          metrics.shouldAssertionsFailedCount}{' '}
+        commands in {metrics.testsFailedCount} tests.
```

Comment:
```suggestion
        {metrics.commandsCount} commands in {metrics.testsFailedCount} tests.
```

This should still be `metrics.commandsCount` based on our discussions. I definitely introduced some confusion here with this [suggestion](https://github.com/w3c/aria-at-app/pull/1288#discussion_r1907848679) so my apologies!

(The `metrics.mustAssertionsFailedCount + metrics.shouldAssertionsFailedCount` in the comment was meant to be an alternative to the `failingAssertions.length` if you preferred, to show how I was thinking of it)